src/main/java/org/fundacionjala/coding/abel/BankOCR.java

```diff
@@ -0,0 +1,139 @@
+package org.fundacionjala.coding.abel;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+/**
+ * Created by AbelBarrientos on 16/5/2017.
+ */
+public final class BankOCR {
+
+    private static final String ZERO  = " _ | ||_|";
+    private static final String ONE   = "     |  |";
+    private static final String TWO   = " _  _||_ ";
+    private static final String THREE = " _  _| _|";
+    private static final String FOUR  = "   |_|  |";
+    private static final String FIVE  = " _ |_  _|";
+    private static final String SIX   = " _ |_ |_|";
+    private static final String SEVEN = " _   |  |";
+    private static final String EIGHT = " _ |_||_|";
+    private static final String NINE  = " _ |_| _|";
+
+    private static final List<Number> ALL_NUMBERS = Arrays.asList(
```

Comment:
This could be simpler
You can use a Map<String, Integer> instead of List<Number>